packages/server/src/threads/automation.ts

```diff
@@ -516,10 +516,8 @@ class Orchestrator {
       filter => {
         Object.entries(filter).forEach(([_, value]) => {
           Object.entries(value).forEach(([field, _]) => {
-            const fromContext = processStringSync(
-              `{{ literal ${field} }}`,
-              this.context
-            )
+            const updatedField = field.replace("{{", "{{ literal ")
```

Comment:
```suggestion
            const updatedField = field.replace("{{", "{{ literal")
```

Reply:
I changed this, but changed it back as I realised we need the space. There needs to be a space after literal for the actual binding i.e `{{ literal steps.1.success }}` instead of `{{ literalsteps.1.success }}`